plugins/generic/customBlockManager/CustomBlockEditForm.inc.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+/**
+ * @file CustomBlockEditForm.inc.php
+ *
+ * Copyright (c) 2003-2015 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * @package plugins.generic.customBlockManager
+ * @class CustomBlockEditForm
+ *
+ * Form for site admins to create and modify sidebar blocks
+ *
+ */
+
+import('lib.pkp.classes.form.Form');
+
+class CustomBlockEditForm extends Form {
+	/** @var $plugin object */
+	var $plugin;
```

Comment:
if those members are private, use the underscore $_plugin to name them. I think you have some cases like that below also.